tests/rule_handling_test.py

```diff
@@ -366,24 +369,20 @@ def passwd_minlen_short_passwd_test(self):
 
         self.ksdata_mock.rootpw.password = "aaaa"
         self.ksdata_mock.rootpw.isCrypted = False
-
-        # run twice --> first run removes the password, but the second one should
-        #               also mention the old password
-        messages = self.rule_data.eval_rules(self.ksdata_mock, self.storage_mock)
         messages = self.rule_data.eval_rules(self.ksdata_mock, self.storage_mock)
 
         # minimal password length greater than actual length --> one warning
         self.assertEqual(len(messages), 1)
-        self.assertEqual(messages[0].type, common.MESSAGE_TYPE_WARNING)
+        self.assertEqual(messages[0].type, common.MESSAGE_TYPE_FATAL)
 
         # warning has to mention the length
         self.assertIn("8", messages[0].text)
 
-        # warning should mention that something happened to the old password
-        self.assertIn("was", messages[0].text)
+        # warning should mention that something is wrong with the old password
+        self.assertIn("is", messages[0].text)
 
-        # doing changes --> password should be cleared
-        self.assertEqual(self.ksdata_mock.rootpw.password, "")
+        # doing changes --> password should not be cleared
+        self.assertEqual(self.ksdata_mock.rootpw.password, "aaaa")
```

Comment:
I don't know - is this correct behaviour? Root PW which does not follow our requirements should be cleared, so that user has to input a new one, right?

Reply:
@dahaic this is no longer the case. The main code no longer clears root password because it breaks the install. See https://bugzilla.redhat.com/show_bug.cgi?id=1263254. This test needs to be changed to reflect that root password is no longer cleared.

Reply:
@redhatrises ah, makes sense. Thanks for finding the reference :)